### PR TITLE
Pin MongoDB and NodeJS version during install

### PR DESCRIPTION
**Pull request by armab**
Rely on major MongoDB and NodeJS versions during the package install.

This is a follow-up for https://github.com/StackStorm/ansible-st2/pull/154 and https://github.com/StackStorm/ansible-st2/pull/157

### CODE REVIEW

File: roles/mongodb/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+# MongoDB version to install
+# Should be '3.2' or '3.4'
+mongodb_version: "3.4"
```
Comment (armab) [inline roles/mongodb/defaults/main.yml]: @cognifloyd this looks like exactly what you asked recently.

Comment (cognifloyd) [reply roles/mongodb/defaults/main.yml]: Yeah. This looks very nice. It's pretty much what I did, though I still used the key directly from mongo.
Hopefully I'll have time soon to push PRs for everything else I fixed (though I don't need to push my mongodb role changes now.) :+1:

**Comment by cognifloyd** [comment]
One more place where the mongo version is hardcoded:
https://github.com/StackStorm/ansible-st2/blob/efa16f3e919dffbdf51fcace46a13e9075f95f34/roles/mongodb/meta/main.yml#L3

**Comment by cognifloyd** [comment]
This needs to be rebased on master to take advantage of the auto testing you just added.

**Comment by armab** [comment]
@cognifloyd good catch, thanks!

File: roles/mongodb/tasks/mongodb_yum.yaml [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+    baseurl: https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/{{ mongodb_major_minor_version }}/x86_64/
+    gpgkey: https://www.mongodb.org/static/pgp/server-{{ mongodb_major_minor_version }}.asc
     state: present
   tags: [databases, mongodb]
 
 - name: yum | Install mongodb
   become: yes
   yum:
-    name: mongodb-org
+    name: mongodb-org-{{ mongodb_version }}*
```
Comment (humblearner) [inline roles/mongodb/tasks/mongodb_yum.yaml]: Shouldn't this be same as: https://github.com/StackStorm/ansible-st2/pull/160/files#diff-7bdf986c5bd89f75152b5fcc42b77680R24 , explicitly install dependencies.

Comment (armab) [reply roles/mongodb/tasks/mongodb_yum.yaml]: No, I checked under CentOS and it pulls correct package version dependencies.

So there is a diff in logic for installing `deb` and `rpm` mongo packages.

**Comment by armab** [comment]
Thanks @cognifloyd and @humblearner for reviewing.
Merging this.

**Pull request merged**